boards/arm/mimxrt595_evk/board.c

```diff
@@ -71,6 +72,11 @@ static int board_config_pmic(const struct device *dev)
 		return ret;
 	}
 
+#if CONFIG_PM
+	/* We can enter deep low power modes */
+	pm_policy_state_lock_put(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+#endif
```

Comment:
ifdeffery not needed, stubs are provided.

Reply:
Thanks, updated.